tests/models/segformer/test_modeling_tf_segformer.py

```diff
@@ -331,64 +329,26 @@ def recursive_check(tuple_object, dict_object):
 
             # todo: incorporate label support for semantic segmentation in `test_modeling_tf_common.py`.
 
+    @unittest.skipIf(
+        not is_tf_available() or len(tf.config.list_physical_devices("GPU")) == 0,
+        reason="TF (<=2.8) does not support backprop for grouped convolutions on CPU.",
```

Comment:
I guess we haven't verified  this with TF 2.9?

Once a new TF version supports this OP on CPU, it's good for us to add a version check inside `skipIf`.

Reply:
👍